.gitignore

```diff
@@ -15,7 +15,8 @@ config.*
 /missing
 /stamp-h1
 /pkgconfig/*.pc
-/m4
+/m4/*
+!/m4/xcam-utils.m4
```

Comment:
maybe you can add another file m4/.gitignore with 
libtool.m4
lt~obsolete.m4
ltoptions.m4
ltsugar.m4
ltversion.m4